.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,86 @@
+name: Release workflow
+# This workflow is triggered on creating tags to master or a opendistro release branch
+on:
+  push:
+  tags:
+    -'v*'
+    branches:
+      - master
+      - opendistro-*
```

Comment:
@qreshi, we'll need to backport this to every other branch for it to be able to run on those right? Or does this only need to be in master, I forget.

Reply:
Yeah, we had to backport our test workflow to older ```opendistro-*``` branches for the jobs to get picked up during PRs, I'm sure it'll be the same case here.